test/inductor/test_flex_attention.py

```diff
@@ -912,14 +912,15 @@ def composed_score_mod(score, b, h, m, n):
         self.run_test(composed_score_mod, dtype)
 
     @supported_platform
+    @skip("Not support compiled flex attention with training bias")
     @common_utils.parametrize("dtype", test_dtypes)
     def test_captured_buffers(self, dtype: torch.dtype):
-        head_offset = torch.rand(H, device="cuda", dtype=dtype)
+        head_offset = torch.rand(8, device="cuda", dtype=dtype, requires_grad=True)
 
         def score_mod(score, b, h, m, n):
             return score + head_offset[h]
 
-        self.run_test(score_mod, dtype)
+        self.run_test(score_mod, dtype, 4, 8, 128, 128)
```

Comment:
Change B, H, S, D to smaller number, since the ```index_put``` is very slow if the indices and values are large.